docs/sources/reference/components/otelcol/otelcol.exporter.debug.md

```diff
@@ -0,0 +1,171 @@
+     -> peer.service: Str(telemetrygen-client)
+Span #1
+    Trace ID       : 3bde5d3ee82303571bba6e1136781fe4
+    Parent ID      :
+    ID             : 5e9dcf9bac4acc1f
+    Name           : lets-go
+    Kind           : Client
+    Start time     : 2023-11-11 04:49:03.50935117 +0000 UTC
+    End time       : 2023-11-11 04:49:03.50949377 +0000 UTC
+    Status code    : Unset
+    Status message :
+Attributes:
+     -> net.peer.ip: Str(1.2.3.4)
+     -> peer.service: Str(telemetrygen-server)
+        {"kind": "exporter", "data_type": "traces", "name": "debug"}"
+```
+
+{{< admonition type="note" >}}
+All instances of `\n`  in the `"detailed"` example have been replaced with new lines.
+{{< /admonition >}}
+
+Setting `use_internal_logger` to `false` is useful if you would like to see actual new lines instead of `\n` in the collector logs.
+However, by not using the internal logger you wouldn't see metadata in the log such as `component_id=otelcol.exporter.debug.default`.
+Multiline logs may also be harder to parse.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.exporter.debug`:
+
+Hierarchy     | Block             | Description                                                                | Required
+--------------|-------------------|----------------------------------------------------------------------------|---------
+debug_metrics | [debug_metrics][] | Configures the metrics that this component generates to monitor its state. | no
+
+The `>` symbol indicates deeper levels of nesting. For example, `client > tls`
+refers to a `tls` block defined inside a `client` block.
+
+[debug_metrics]: #debug_metrics-block
+
+### debug_metrics block
+
+{{< docs/shared lookup="reference/components/otelcol-debug-metrics-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components:
+
+Name | Type | Description
+---- | ---- | -----------
+`input` | `otelcol.Consumer` | A value that other components can use to send telemetry data to.
+
+`input` accepts `otelcol.Consumer` data for any telemetry signal (metrics,
+logs, or traces).
+
+## Component health
+
+`otelcol.exporter.debug` is only reported as unhealthy if given an invalid
+configuration.
+
+## Debug information
+
+`otelcol.exporter.debug` does not expose any component-specific debug
+information.
+
+## Example
+
+This example scrapes Prometheus UNIX metrics and writes them to the console:
+
+```river
+prometheus.exporter.unix "default" { }
+
+prometheus.scrape "default" {
+    targets    = prometheus.exporter.unix.default.targets
+    forward_to = [otelcol.receiver.prometheus.default.receiver]
+}
+
+otelcol.receiver.prometheus "default" {
+    output {
+        metrics = [otelcol.exporter.debug.default.input]
+    }
+}
+
+otelcol.exporter.debug "default" {
+    verbosity           = "detailed"
+    sampling_initial    = 1
+    sampling_thereafter = 1
+}
+```
```

Comment:
nit: I would suggest to put a simpler example (otlp receiver -> debug exporter). Having a convert from OTLP to Prometheus as example is not ideal because it's usually preferable to avoid converting the data when possible. Examples should follow the most common use cases

Reply:
Fair enough, I changed the example to be simper :)